src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Grid\Definition\Factory;
+
+use PrestaShop\PrestaShop\Core\Grid\Action\GridActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShopBundle\Form\Admin\Type\DateRangeType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class LogGridDefinitionFactory is responsible for creating new instance of Log grid definition
+ */
+final class LogGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    /**
+     * @var EngineInterface
+     */
+    private $templating;
+
+    /**
+     * @param EngineInterface $templating
+     */
+    public function __construct(EngineInterface $templating)
+    {
+        $this->templating = $templating;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getIdentifier()
+    {
+        return 'logs';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return $this->trans('Logs', [], 'Admin.Advparameters.Feature');
```

Comment:
Is the string 'Logs by email' working here?

Reply:
maybe but this has no sense regarding what the grid represents: what we are naming here is the table of logs available in Configure > Advanced Parameters > Logs

Reply:
Okay, thanks Mickaël, then it should be in domains Admin.Navigation.Menu.

Reply:
fixed! thank you :)